.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
```

Comment:
I'm not familiar with dependabot at all -- can you tell me how to test this (you mention needing to make sure to add it to the repo?) either in the conversation for the PR or as an addendum to Contributing.md?

Reply:
Dependabot is a bot that automatically opens PR to this repository when an action in CI has an updated version (pushes to its own branch in repo, thus needs write access).

Reply:
> when an action in CI has an updated version

This is the bit I don't understand -- I'm used to using Github's CI tools to check if a branch passes tests before merging a PR.

What workflow are you automating here?